sklearn/covariance/shrunk_covariance_.py

```diff
@@ -439,9 +439,8 @@ def oas(X, assume_centered=False):
     where mu = trace(cov) / n_features
 
     The formula we used to implement the OAS
-    does not correspond to the one given in the article. It has been taken
-    from the MATLAB program available from the author's webpage
-    (http://tbayes.eecs.umich.edu/yilun/covestimation).
+    does not correspond to the one given in the article. It is slightly
+    modified (see OAS class description for details).
```

Comment:
You can use a sphinx directive here, this way this will be a link in the HTML doc, example usage:
```
Use :class:`sklearn.model_selection.ParameterSampler` instead.
```

Look at `git grep :class:` for more examples